about.md

```diff
@@ -198,13 +45,31 @@ Participants join this program with a project that they either are already worki
 Project ideas can range from solving technical questions to creating an open data project or report, developing an open source software project, writing an open publication, facilitating community/team culture movements, advancing open educational resources or contributing to other existing projects/communities.
 
 **Check out the projects developed in the previous cohorts:**
-- [{{ ols-1-projects | size }} projects](/ols-1/projects-participants#projects) ([{{ ols-1-participants | size }} participants](/ols-1/projects-participants#participants)) for [**OLS-1**]({% link ols-1.md %}) ({{ ols-1-schedule.weeks['01'].start }} - {{ ols-1-end }})
-- [{{ ols-2-projects | size }} projects](/ols-2/projects-participants#projects) ([{{ ols-2-participants | size }} participants](/ols-2/projects-participants#participants)) for [**OLS-2**]({% link ols-2.md %}) ({{ ols-2-schedule.weeks['01'].start }} - {{ ols-2-end }})
-- [{{ ols-3-projects | size }} projects](/ols-3/projects-participants#projects) ([{{ ols-3-participants | size }} participants](/ols-3/projects-participants#participants)) for [**OLS-3**]({% link ols-3.md %}) ({{ ols-3-schedule.weeks['01'].start }} - {{ ols-3-end }})
-- [{{ ols-4-projects | size }} projects](/ols-4/projects-participants#projects) ([{{ ols-4-participants | size }} participants](/ols-4/projects-participants#participants)) for [**OLS-4**]({% link ols-4.md %}) ({{ ols-4-schedule.weeks['01'].start }} - {{ ols-4-end }})
-- [{{ ols-5-projects | size }} projects](/ols-5/projects-participants#projects) ([{{ ols-5-participants | size }} participants](/ols-5/projects-participants#participants)) for [**OLS-5**]({% link ols-5.md %}) ({{ ols-5-schedule.weeks['01'].start }} - {{ ols-5-end }})
-- [{{ ols-6-projects | size }} projects](/ols-6/projects-participants#projects) ([{{ ols-6-participants | size }} participants](/ols-6/projects-participants#participants)) for [**OLS-6**]({% link ols-6.md %}) ({{ ols-6-schedule.weeks['01'].start }} - {{ ols-6-end }})
-- [{{ ols-7-projects | size }} projects](/ols-7/projects-participants#projects) ([{{ ols-7-participants | size }} participants](/ols-7/projects-participants#participants)) for [**OLS-7**]({% link ols-7.md %}) ({{ ols-7-schedule.weeks['01'].start }} - {{ ols-7-end }})
+{% for cohort_folder in cohorts %}
+{% assign cohortName = cohort_folder[0] %}
```

Comment:
Please indent here

Reply:
It does work...and you're right. This is much cleaner. Thanks. 👍

Reply:
To be sure please, you mean indent this line 👇 everywhere in the file?
`{% assign cohort_name = cohort[0] %}`

I try to avoid indenting the lines with `assign` because that usually brings me to this type of situation...

![Screenshot (549)](https://github.com/open-life-science/open-life-science.github.io/assets/105166953/ed3422d1-58d2-4fae-8062-3fe7a8bc1632)

Reply:
I made changes so it gets indented

Reply:
Ah. Jekyll is very strange. Sometimes, it's adding spaces between lines that's the problem. Other times, it's handling whitespace. 

Thank you so much, Bérénice. I looked at the preview as well. Looks good!